.ci/test-local-tap.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eo pipefail
+
+. "${0%/*}/libfunctions.sh"
+
+echo '--- :evergreen_tree: Configuring Environment'
+exec 9>&1 # enable tee to write to STDOUT as a file
+test_bottle
+
+echo '+++ :beer: Homebrew Tap and Install'
+perform 'brew update'
+if [[ ! $(python3 --version 2>/dev/null) ]]; then
+   perform 'brew install python3'
+fi
```

Comment:
**UX** - Can we move this `brew install python3` step out of the test log folding group and back into the `Configuring Environment` log folding group? It can be done before a `brew update`, and I don't think the output of this command is useful to the customer or relevant to the test. I guess we could also suppress this, but moving it would be preferred because it is just noise. Up to you.

Reply:
Can, if that's the case, I think moving `brew update` and maybe other setup calls would be fair game also.

Reply:
Yes, I agree.